packages/extension-ui/src/Popup/Authorize/index.tsx

```diff
@@ -4,48 +4,50 @@
 import React, { useContext } from 'react';
 import styled from 'styled-components';
 
-import { AuthorizeReqContext } from '../../components';
+import { AuthorizeReqContext, Loading, RequestPagination, ScrollWrapper } from '../../components';
+import useRequestsPagination from '../../hooks/useRequestsPagination';
+import useTranslation from '../../hooks/useTranslation';
 import Request from './Request';
 
 function Authorize(): React.ReactElement {
+  const { t } = useTranslation();
   const requests = useContext(AuthorizeReqContext);
-
-  return (
-    <Container>
-      {requests.map(
-        ({ id, request, url }, index): React.ReactNode => (
-          <Request
-            authId={id}
-            isFirst={index === 0}
-            key={id}
-            request={request}
-            url={url}
+  const { index: requestIndex, next, previous, request } = useRequestsPagination(requests);
+
+  return request ? (
```

Comment:
I think sth here or in `Request.tsx` broke this screen:
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/e4fd4585-f20a-4894-bdbf-74198aaebe0f)